postman_tests/lib/es.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import absolute_import
+import os
+import glob
+import json
+import csv
+import requests
+from . import common
+
+
+def get_es_api_url(config_path):
+    if os.path.exists(f"{config_path}/dev.json"):
+        config_file_path = f"{config_path}/dev.json"
+    else:
+        config_file_path = f"{config_path}/prod.json"
```

Comment:
Why do we need two different things for running tests ?

Reply:
One is for configuration  for devs the other is for running in CI